src/lib/components/Cytoscape.react.js

```diff
@@ -630,7 +643,8 @@ Cytoscape.defaultProps = {
     autolock: false,
     autoungrabify: false,
     autounselectify: false,
-    autoRefreshLayout: true
+    autoRefreshLayout: true,
+    responsive: false
```

Comment:
Should `responsive` be true by default instead?

Reply:
I chose to use `responsive: false` so that existing default functionality will remain unchanged and existing `dash-cytoscape` apps will not be unexpectedly altered by the new functionality. However if this is not an issue I can change it to `responsive: true`. Just let me know!

Reply:
You are right. In this case, let's keep it `responsive: false`, but in the readme.md, let's 
* add a subsection (right below "External layouts") that shows how activate it
* make sure to have an `usage-resize.py` file in `demos` that shows how to resize
* make a gif and add it at the end of the Gallery section in readme